lib/fastlane/plugin/icon_versioning/helper/version_icon_helper.rb

```diff
@@ -134,7 +135,7 @@ def version_icon(original_icon_path, versioned_icon_path)
         MiniMagick::Tool::Convert.new do |convert|
           convert << '-background' << 'none'
           convert << '-size' << "#{image_width - (text_left_margin + text_right_margin)}x#{band_height - (text_top_margin + text_bottom_margin)}"
-          convert << '-fill' << 'white'
+          convert << '-fill' << "#{@color}"
```

Comment:
You could simply use `@color` instead of `"#{@color}"`.

Reply:
Fair point, done